build.sbt

```diff
@@ -23,4 +62,11 @@ lazy val icenet = project.settings(commonSettings).dependsOn(rocketchip, testchi
 
 lazy val boom = project.settings(commonSettings).dependsOn(rocketchip)
 
-lazy val example = (project in file(".")).settings(commonSettings).dependsOn(boom, icenet, testchipip, sifive_blocks)
+// The library components of FireSim
+lazy val midas   = ProjectRef(firesimDir, "midas")
+lazy val firesim = ProjectRef(firesimDir, "common")
+
+lazy val firechip = (project in file("."))
+    .settings(commonSettings)
+    .dependsOn(boom, icenet, testchipip, sifive_blocks, midasTargetUtils, midas,
+      firesim % "test->test;compile->compile")
```

Comment:
Is the test dependency needed? I assume so, otherwise you wouldn't bother, but want to check in case this is no longer true after refactoring.

Reply:
There are common classes in test i extend in tests i've moved into rebar. I could make the dependency more specific.

Reply:
There are common test classes i extend here -- i should be able to make the dependency far more percise as it's more test:compile->test:compile right? I already forget the scope syntax.

Reply:
You could make the dep more precise but it doesn't really matter that much for test.